actions/index.ts

```diff
@@ -1,9 +1,21 @@
-export const addItems = (data: string) => {
+export const addItems = (
+  taskTitleData: string,
+  descriptionData: string,
+  startDateData: string,
+  endDateData: string,
+  priorityData: string,
+  statusData: string
+) => {
   return {
     type: "ITEM_ADDED",
     payload: {
       id: new Date().getTime().toString(),
```

Comment:
Well this could be good for front-end but not a nice practice on backend as it could produce bugs.